app/javascript/src/infinite-scroll.js

```diff
@@ -64,6 +64,20 @@ function getInfiniteScrollContent(element) {
         element.innerHTML = "Load more"
         element.setAttribute("data-url", nextUrl)
       }
+    },
+    error: (error) => {
+      progressBar.setValue(1)
+      progressBar.hide()
+
+      const spinner = document.querySelector(".items").querySelector(".spinner")
```

Comment:
Let's add some optional chaining here, just in case.

`document.querySelector(".items")?.querySelector(".spinner")`